recipe/meta.yaml

```diff
@@ -5,28 +5,29 @@ package:
   version: {{ version }}
 
 source:
-  - url: https://github.com/KhronosGroup/SPIRV-Tools/archive/refs/tags/v{{ version }}.tar.gz
-    sha256: 7416cc8a98a10c32bacc36a39930b0c5b2a484963df5d68f388ed7ffee1faad3
-
-  - url: https://github.com/KhronosGroup/SPIRV-Headers/archive/refs/tags/sdk-1.3.243.0.tar.gz
-    sha256: 16927b1868e7891377d059cd549484e4158912439cf77451ae7e01e2a3bcd28b
-    folder: external/spirv-headers
+  url: https://github.com/KhronosGroup/SPIRV-Tools/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 7416cc8a98a10c32bacc36a39930b0c5b2a484963df5d68f388ed7ffee1faad3
+  patches:
+    - 0001-Find-Python3-better.patch
```

Comment:
Is this needed?

Reply:
happy to try without it. i think the issue was the git stuff that was tripping me.